gaffer-as-a-service/gaas-rest/src/main/resources/config/accumulo.yaml

```diff
@@ -0,0 +1,2 @@
+accumulo:
+  enabled: true
```

Comment:
I think it is best to have a newline at the end of yaml files. This applies a few times in the PR.

Reply:
Done.